SPECS-EXTENDED/distribution-gpg-keys/distribution-gpg-keys.spec

```diff
@@ -47,7 +47,7 @@ cp -a keys/* %{buildroot}%{_datadir}/%{name}/
 
 %changelog
 * Wed Jan 5 2022 Cameron Baird <cameronbaird@microsoft.com>  - 1.60-2
-- Add to SPECS-EXTENDED from Fedora
+- Initial CBL-Mariner import from Fedora 33 (license: CC0)
```

Comment:
This line should have the license of the spec file, which should be MIT for Fedora specs.

Reply:
Uh oh, I may have copy-pasted the wrong licenses...

Reply:
@thcrain-msft 
The license for this spec actually is CC0, see https://github.com/xsuchy/distribution-gpg-keys/blob/main/LICENSE

Reply:
Ah! In that case, since you're pulling the spec file from the source repo, you should credit the source repo instead of Fedora here.

Reply:
The spec file contributed to Fedora is MIT unless the spec file says otherwise.

Reply:
I've resolved a lot of confusion regarding how we handle licenses (when to use spec vs src license) in a talk with Pawel. If you recheck my work, things should hopefully line up.